pkg/processor/runtime/python/py/_nuclio_wrapper.py

```diff
@@ -35,6 +36,9 @@ class Constants:
     # in msgpack protoctol, binary messages' length is 4 bytes long
     msgpack_message_length_bytes = 4
 
+    drain_signal = signal.SIGUSR1
```

Comment:
```suggestion
    drain_signal = signal.SIGUSR2
```

Reply:
Thanks @rokatyy. This is what I get for making changes on a the fly while still on a call...